scripts/library.sh

```diff
@@ -445,7 +445,15 @@ function get_canonical_path() {
 # Return the base url we use to build the actual knative yaml sources.
 function get_knative_base_yaml_source() {
   local knative_base_yaml_source="https://storage.googleapis.com/knative-nightly/@/latest"
-  local branch_name="$(git rev-parse --abbrev-ref HEAD)"
+  local branch_name=""
```

Comment:
Join the two assignments, drop the overcomplicated comment; just mention that we get the branch from Prow's env var, by default.